sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectSummary.groovy

```diff
@@ -0,0 +1,74 @@
+package life.qbic.portal.sampletracking.components.projectoverview
+
+import groovy.transform.EqualsAndHashCode
+import life.qbic.datamodel.dtos.projectmanagement.Project
+
+/**
+ * <b>Project Summary POJO</b>
+ *
+ * <p>A simple java object to be used in the project overview grid.</p>
+ * <p>This class hold information to be shown to the user concerning a specific project.</p>
+ *
+ * @since 1.0.0
+ */
+@EqualsAndHashCode(includeFields = true)
+class ProjectSummary {
+    String code
+    String title
+    int samplesReceived
```

Comment:
The question is: do we want this class to be mutable?

Reply:
yes we have to. It is used in the grid and modified where data is updated

Reply:
👍🏽  thanks foclarifyingng this